debian/ubuntu-advantage-tools.prerm

```diff
@@ -3,12 +3,14 @@
 set -e
 
 remove_apt_files() {
-    /usr/bin/python3 -c '
-from uaclient.apt import clean_apt_files
-
-clean_apt_files()
-'
-
+    # This list should be kept up to date with the list of available apt-repo-based services
+    for service in anbox cc-eal cis esm-apps esm-infra fips fips-preview fips-updates realtime-kernel ros ros-updates; do
```

Comment:
I think we also need `usg` here

Reply:
It won't hurt, but is the apt source file named with the `presented_as` name?

Reply:
nope, source-file and key will be `ubuntu-cis.list` unfortunately :clown_face: :upset_cry: